docs/docs/features/splits.mdx

```diff
@@ -0,0 +1,180 @@
+Dongles are usually connected to the host device using USB. This way the central half of the keyboard can be powered by the host device, 
+and the peripheral halves can be powered by their own batteries. This setup allows both halves of the keyboard to have a longer battery life.
+
+
+![Mermaid Connection](../assets/splits/mermaid-connection.png)
+
+{/* Here is an example diagram of a split keyboard with a dongle:
+
+```mermaid
+graph TD;
+    A[Computer/Host Device] <-- USB or BLE --> DONGLE[Dongle/Central];
+    DONGLE -. BLE .-> LEFT[Left Half/Peripheral];
+    DONGLE -. BLE .-> RIGHT[Right Half/Peripheral];
+```
+
+Dotted lines represent BLE connections.
+Solid lines represent USB connections. */}
+
+:::info
+For more information on how to set up a two split keyboard, refer to [New Keyboard Shield](../development/new-shield.mdx) under Guides.
+After setting up the split keyboard, you can follow the steps below to add a dongle to the setup.
+:::
+
+## Benefits/Disadvantages of using a Dongle
+
+Benefits:
+
+- Adding a dongle, enables both keyboard halves to act as peripherals, meaning equal battery life for both halves.
+- Easier to connect to the host device.
+
+Disadvantages:
+
+- More complex setup.
+- Extra microcontroller needed
+- Keyboard is unusable without the dongle.
+- Added latency due to the extra hop between the host device and the keyboard halves.
+
+
+
+## Defining a Dongle for a Keyboard 
+
+
+:::note
+In the examples below, we will be referencing the New Split Keyboard Shield guide files and will be building on top of that.
+:::
+
+### Keyboard Shield
+
+First we will introduce a third split to our keyboard configuration. This will be used as the dongle.
+
+```kconfig title="Kconfig.shield"
+config SHIELD_MY_BOARD_DONGLE
+    def_bool $(shields_list_contains,my_board_dongle)
+
+config SHIELD_MY_BOARD_LEFT
+    def_bool $(shields_list_contains,my_board_left)
+
+config SHIELD_MY_BOARD_RIGHT
+    def_bool $(shields_list_contains,my_board_right)
+```
+
+### Keyboard Defconfig
+
+Next we will define the roles of the keyboard halves. The left and right halves will be set as peripherals, and the dongle will be set as the central.
+We will also only give the central device the keyboard name.
+
+For the dongle configuration we also want to specify that we are expecting 2 peripherals, one for the left half and one for the right half.
+We will also set the `ZMK_SLEEP` to `n` to prevent the dongle from going to sleep. 
+If the dongle is sleeping, keystrokes will not be sent to the host device, until the dongle wakes up.
+
+Finally, we will set the `BT_CLTR_TX_PWR_PLUS_8` to `y` to increase the transmit power of the dongle.
+Normally this would drain the battery faster, but since the dongle is connected to the host device, it will not be a problem.
+
+There is also an option to still use the left half as the central, and the right half as the peripheral. In that case the dongle will not be used.
```

Comment:
I think this would be good to describe in a bit more detail. See the comment following this one.

Reply:
Reworked it